.env.example

```
@@ -30,14 +30,22 @@ DOMAIN_NAME=example.com
+LETSUR_AI_GATEWAY_API_KEY=your-letsur-ai-gateway-api-key
 # LIGHT: 선별용(경로/chunk/문서 선택), NORMAL: 단순 응답, HEAVY: 최종 답변 생성
+LETSUR_AI_GATEWAY_MODEL_LIGHT=gpt-4o-mini
+LETSUR_AI_GATEWAY_MODEL_NORMAL=gpt-4o-mini
+LETSUR_AI_GATEWAY_MODEL_HEAVY=gpt-4o-mini
+
+# Legacy: Open Router
+# 기존 OpenRouter 연동용 환경변수입니다. Letsur AI Gateway 전환 후에는 기본적으로 사용하지 않습니다.
+OPEN_ROUTER_API_KEY=your-openrouter-api-key
 OPEN_ROUTER_MODEL_LIGHT=google/gemini-2.0-flash-001
 OPEN_ROUTER_MODEL_NORMAL=google/gemini-2.0-flash-001
 OPEN_ROUTER_MODEL_HEAVY=openai/gpt-4o
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OPEN_ROUTER_MODEL_HEAVY key should go before the OPEN_ROUTER_MODEL_LIGHT key

(UnorderedKey)